pkgs/applications/graphics/odaconverter/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, mkDerivation, dpkg, fetchurl, qtbase }:
+
+let
+version = "21.7.0.0";
+rpath = "$ORIGIN:${lib.makeLibraryPath [ stdenv.cc.cc qtbase ]}";
+
+in mkDerivation {
+  pname = "oda-converter";
+  inherit version;
```

Comment:
Please add a `name` attribute, such as:
```
name = "${pname}-${version}";
```

Reply:
Is it necessary? I thought `pname` + `version` was all you needed?

Reply:
Seems like you're right, sorry. [This section](https://nixos.org/nixpkgs/manual/#sec-package-naming) of the nixpkg manual made it seem like it is required, but it isn't.